ccd-definition/CaseTypeTab/CareSupervision/11_C2-nonprod.json

```diff
@@ -17,7 +17,17 @@
     "TabID": "OtherApplicationsTab",
     "TabLabel": "Other applications",
     "TabDisplayOrder": 11,
-    "CaseFieldID": "c2DocumentBundle",
+    "CaseFieldID": "additionalApplicationsBundle",
     "TabFieldDisplayOrder": 2
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "OtherApplicationsTab",
+    "TabLabel": "Other applications",
+    "TabDisplayOrder": 11,
+    "CaseFieldID": "c2DocumentBundle",
+    "TabFieldDisplayOrder": 3
```

Comment:
existing cases with old c2 data should appear at the bottom of the tab (discussed with UX)